apps/front-client/src/features/sidebar/components/Molecules/Profile.tsx

```diff
@@ -25,6 +27,21 @@ const ProfileDescription = styled(Box)`
 `;
 
 const Profile = () => {
+  const session = useSession();
+  const user = session?.data?.user;
+
+  if (!user) {
+    return <p>User not found or not logged in</p>;
+  }
+
+  const getFullName = () => {
+    return `${user.first_name || ''} ${user.last_name || ''}`.trim();
+  };
+
+  const getEmail = () => {
+    return user.email || 'Email not provided';
+  };
```

Comment:
Pourquoi c'est des fonctions ?
(Vraiment nécessaire le `.trim()` ? 🤔 
```suggestion
  const fullname = `${user.first_name} ${user.last_name}`.trim();
  const email = user.email || 'Email not provided';
```